02-peer-protocol.md

```diff
@@ -936,6 +936,8 @@ change the commitment transaction aside from the new revocation hash
 fee changes).
   - MUST include one `htlc_signature` for every HTLC transaction corresponding
   to BIP69 lexicographic ordering of the commitment transaction.
+  - if it has not recently received a message from the remote node:
+      - SHOULD use `ping` and await the reply `pong` before sending `commitment_signed`.
```

Comment:
Can we also say alternatively that a node should regularly ping its peers? That's easier to implement and achieves almost the same goal.